offscreen/offscreen.html

```diff
@@ -5,10 +5,6 @@
     <title>MetaMask Offscreen Page</title>
   </head>
   <body>
-    <script src="./load-offscreen.js"></script>
-    <iframe
-      src="https://metamask.github.io/eth-ledger-bridge-keyring"
-      allow="hid"
-    ></iframe>
+    <script type="text/javascript" src="./load-offscreen.js"></script>
```

Comment:
This should not have a type field.

> Authors should omit the [type](https://html.spec.whatwg.org/dev/scripting.html#attr-script-type) attribute instead of redundantly setting it.

https://html.spec.whatwg.org/dev/scripting.html#attr-script-type

Reply:
Can we do this in a separate PR as it's unrelated to the change here and purpose?